### PR TITLE
Add annotation for SNO installs

### PR DESCRIPTION
**Pull request by gnufied**
xref https://bugzilla.redhat.com/show_bug.cgi?id=2070194

cc @lpettyjo

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@lpettyjo, PTAL

File: modules/persistent-storage-local-install.adoc [hunk 1]
```diff
@@ -34,6 +34,27 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
 $ oc annotate project openshift-local-storage openshift.io/node-selector=''
 ----
 
+. Optional: Allow local-storage operator installation on single node clusters.
++
+To allow installation of local-storage operator in single node clusters, we need to add following annotation to the namespace in which operator is being installed.
```
Comment (lpettyjo) [inline modules/persistent-storage-local-install.adoc:39]: Change to: To allow installation of local storage operator in single node clusters, add the following annotation to the namespace in which operator is being installed:

File: modules/persistent-storage-local-install.adoc [hunk 2]
```diff
@@ -34,6 +34,27 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
...
++
+[source,terminal]
+----
+$ oc annotate project openshift-local-storage workload.openshift.io/allowed='management'
+----
++
+Additionally the annotation can also be added by editing `openshift-local-storage` namespace:
```
Comment (lpettyjo) [inline modules/persistent-storage-local-install.adoc:46]: Change to: Additionally, you can add the annotation by editing `openshift-local-storage` namespace:

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Pull request closed without merging**